DEV_GUIDE.md

```diff
@@ -457,6 +490,92 @@ Certificate of Origin (DCO)](./DCO.txt).
 This can be done using `git commit -s` for each commit
 in your pull request. Alternatively, to signoff a bunch of commits you can use `git rebase --signoff _your-branch_`.
 
+# Development Guide for Kroxylicious Operator
+
+This is the development guide for Kroxylicious operator for Kubernetes.
+
+## Hacking and Debugging
+
+If you want to iterate quickly on the operator the simplest way is to run it as a process on your host (i.e. *not* running it within a Kubernetes cluster).
+
+Note: The Integration Tests will only run if your kubectl context is pointing at a cluster. For development, we recommend using  `minikube`, for example:
+
+```bash
+minikube start --kubernetes-version=latest --driver=podman 
+````
+
+You should now be able to run the tests using `mvn`.
+
+If you want to run the `OperatorMain` (e.g. from your IDE, maybe for debugging) then you'll need to install the CRD:
+
+```bash
+kubectl apply -f ../kroxylicious-kubernetes-api/src/main/resources/META-INF/fabric8
+```
+
+You should now be able to play around with `KafkaProxy` CRs; read the "Creating a `KafkaProxy`" section.
+
+Alternatively you can build the operator properly and run it within Kube...
+
+## Building the operator
+
+Refer to section [Building and pushing a Kroxylicious Container Image](#building-and-pushing-a-kroxylicious-container-image).
+
+## Installing the operator
+
+Spin up a minikube custer:
+
+```bash
+minikube start --kubernetes-version=latest --driver=podman 
+````
+
+## Installing the operator
+
+```bash
+kubectl apply -f kroxylicious-operator/target/packaged/install 
+```
+
+You can check that worked with something like
+
+```bash
+kubectl logs -n kroxylicious-operator pods/kroxylicious-operator-7cd88454c8-fjcxm operator
```

Comment:
This was written before the CRs reported their own status.
Let's switch this for a command query the kafkaproxy CR.